packages/tiled-grid/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@automattic/tiled-grid",
+  "private": "true",
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/runtime": "~7.2.0",
+    "concurrently": "~4.1.0",
+    "nodemon": "~1.18.8"
```

Comment:
Review deps and scripts when main development work is finished

Reply:
Removed package.json when moving to the block folder.